deb/brave-keyring/etc/sysctl.d/brave.conf

```diff
@@ -0,0 +1,3 @@
+# Enable user namespaces which are required by the Chromium sandbox
```

Comment:
Do we want this file to be prepended by a number to match the existing files on Fedora/Ubuntu?

Reply:
I think that's only important if the order of processing is important. In our case, it really doesn't matter.